infrastructure/ecs_task/outputs.tf

```diff
@@ -32,4 +32,19 @@ output "trigger_dead_letter_queue" {
 
 output "results_dead_letter_queue" {
   value = module.taskmodule.results_dead_letter_queue
-}
\ No newline at end of file
+}
+
+output "task_queue_consumer_arn" {
+  value = module.taskmodule.task_queue_consumer_arn
+}
+
+output "task_queue_consumer_role" {
+  value = module.taskmodule.task_queue_consumer_role
+}
+
+output "s3_bucket_policy_arn" {
+  value = module.taskmodule.s3_bucket_policy_arn
+}
+output "s3_bucket_policy_doc" {
```

Comment:
This is not used, probably not in the task module either, can we clean both up?

Why do you need the bucket policy or the task queue consumer role to be passed out of here? The bucket policy belongs with the bucket creation.

Reply:
removed